deploy/secrets-store-csi-driver-windows.yaml

```diff
@@ -0,0 +1,78 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-secrets-store
+spec:
+  selector:
+    matchLabels:
+      app: csi-secrets-store
+  template:
+    metadata:
+      labels:
+        app: csi-secrets-store
+    spec:
+      nodeSelector:
+        beta.kubernetes.io/os: windows
+      serviceAccountName: secrets-store-csi-driver
+      containers:
+        - name: node-driver-registrar
+          image: mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v1.3.0-pipeline
+          args:
+            - --v=5
+            - "--csi-address=unix://C:\\csi\\csi.sock"
+            - "--kubelet-registration-path=C:\\var\\lib\\kubelet\\plugins\\csi-secrets-store\\csi.sock"
+          env:
```

Comment:
Is there an equivalent command we can run to remove the socket for preStop?
```
rm -rf /registration/secrets-store.csi.k8s.io-reg.sock
```

Reply:
Will try it out and see if we can use a windows command to remove it

Reply:
Added the prestop command to delete the socket